locales/ruRU.lua

```diff
@@ -3110,7 +3110,7 @@ for key,value in pairs({
 	[17] = {4, "|CFF00FFDEДля этого требуется Лунный Фестиваль.|r", "Лунный Фестиваль"},
 	[22] = {4, "|CFF00FFDEДля этого требуется определенный микро-праздник.|r", "Маленькие праздники"},
 	[21] = {4, "|CFF00FFDEДля этого требуется Огненный Солнцеворот.|r", "Огненный Солнцеворот"},
-	[1009] = {4, "|CFF00FFDEДля этого требуется праздник празднования Нового года.|r", "New Years Celebration" },
+	[1009] = {4, "|CFF00FFDEДля этого требуется Канун Нового года.|r", "Отмечаем Новый Год" },
```

Comment:
```suggestion
	[1009] = {4, "|CFF00FFDEДля этого требуется Празднество фейерверков.|r", "Празднество фейерверков" },
```
I'd still prefer it to be a part of Winter Veil but let's use the name from the calendar. 🙄

Reply:
But this one I actually took from wowhead https://www.wowhead.com/quest=8860/new-year-celebrations, because it is New Year's Eve event there.

Reply:
You are looking at the quest name though, I meant calendar event name.
https://www.wowhead.com/event=658/fireworks-celebration